core-web/libs/dotcms-scss/angular/dotcms-theme/components/_table.scss

```diff
@@ -0,0 +1,28 @@
+@use "variables" as *;
+p-table {
+    .p-datatable {
+        border: 1px solid $color-palette-gray-400;
+        border-radius: $border-radius-md;
+    }
+
+    .p-datatable > .p-datatable-wrapper {
+        border-radius: $border-radius-md;
+    }
+
+    .p-datatable .p-datatable-thead > tr > th {
+        background-color: $color-palette-gray-200;
+        color: $black;
```

Comment:
should we use nested rules instead of `>`?

what do you think? @fmontes

Reply:
Good question... I'll guess it depends of what you are doing here.